internal/pkg/service/buffer/store/configstore/configstore.go

```diff
@@ -35,48 +34,6 @@ func New(logger log.Logger, etcdClient *etcd.Client, validator validator.Validat
 	return &Store{logger, etcdClient, validator, tracer}
 }
 
-func ReceiverKey(projectID int, receiverID string) string {
-	return fmt.Sprintf("config/receiver/%d/%s", projectID, receiverID)
-}
-
-func ReceiverPrefix(projectID int) string {
-	return fmt.Sprintf("config/receiver/%d", projectID)
-}
-
-func ExportsPrefix(projectID int, receiverID string) string {
-	return fmt.Sprintf("config/export/%d/%s/", projectID, receiverID)
-}
-
-func ExportKey(projectID int, receiverID string, exportID string) string {
-	return fmt.Sprintf("config/export/%d/%s/%s", projectID, receiverID, exportID)
-}
-
-func MappingsPrefix(projectID int, receiverID string, exportID string) string {
-	return fmt.Sprintf("config/mapping/revision/%d/%s/%s/", projectID, receiverID, exportID)
-}
-
-func MappingKey(projectID int, receiverID string, exportID string, revisionID int) string {
-	return fmt.Sprintf("config/mapping/revision/%d/%s/%s/%08d", projectID, receiverID, exportID, revisionID)
-}
```

Comment:
Preco to cele robim, ... pri implementacii `Worker` my isla :exploding_head: z etcd keys.
Tu je to este celkom easy, lebo to je viazane na project/receiver... no ak potrebuje vsetky receivers pre worker, ... alebo vsetky exports z projektu, tak sa to zacina komplikovat a bolo by tu strasne vela funkcii na generovanie klucov a velke riziko chyby.